server/utils/helperFunctions.js

```diff
@@ -0,0 +1,146 @@
+/*
```

Comment:
I like that you considered the validation on the incoming inputs - it's excellent to see. However, I don't want to have to maintain helpers like this.

Rather, I think it would make much more sense to utilize a library like https://express-validator.github.io/docs/ which is already being used in the server so I just want to keep it consistent.

Code example: https://github.com/hatchways/team-oak/blob/main/server/validate.js

Reply:
Got this done. I'm not going to push the changes until I have a chance to double check and make sure migrating to middlewear works as intended.